retailer-oas/retailer.yaml

```diff
@@ -389,39 +393,18 @@ paths:
         404:
           description: Picking Location not found
 
-  # Delivery Areas
-  /delivery-areas/{id}:
-    get:
-      summary: Find a Delivery Area by identifier
-      description: Get a single Delivery Area for a given identifier.
-      operationId: findDeliveryAreaById
-      tags:
-        - delivery-areas
-      parameters:
-        - name: id
-          in: path
-          description: Delivery Area identifier
-          required: true
-          schema:
-            type: string
-      responses:
-        200:
-          description: OK
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/DeliveryArea'
+  # Timeslots
 
-  /delivery-areas/{id}/timeslots:
+  /timeslots:
     get:
       summary: Get the Time Slots and their availability for a given Delivery Area
-      description: Returns the delivery Time Slots available for a given Delivery Area and date.
+      description: This endpoint allows retailers to retrieve the available time slots for a given delivery area. The retailer can specify the delivery area by passing its identifier in the id query parameter. The fromDate parameter is used to specify the starting date of the time slots to be retrieved. The date must be in ISO 8601 format. The quantity parameter is used to limit the number of time slots to be returned. This parameter is useful to limit the request to retrieve just the nearest available time slots.
```

Comment:
Suggestion:

This endpoint allows retailers to retrieve the available time slots for a given delivery area. The retailer can specify the delivery area by passing its identifier in the `id` query parameter. The `fromDate` parameter is used to specify the starting date of the time slots to be retrieved. The date must be in ISO 8601 format. The `quantity` parameter limits the number of time slots to be returned. Use this parameter to limit the request and retrieve just the nearest available time slots.